cloud-sql/mysql/servlet/.env.yaml

```
@@ -0,0 +1,7 @@
+INSTANCE_CONNECTION_NAME:
+INSTANCE_UNIX_SOCKET:
+DB_USER:
+DB_PASS:
+DB_NAME:
+INSTANCE_HOST:
+INSTANCE_PORT:
```

nit: maybe prompt users what to fill in? This is the wording we've been using in Go & Python (DB_PORT vs INSTANCE_PORT)
```suggestion
INSTANCE_CONNECTION_NAME: <PROJECT-ID>:<INSTANCE-REGION>:INSTANCE-NAME>
INSTANCE_UNIX_SOCKET: /cloudsql/<PROJECT-ID>:<INSTANCE-REGION>:INSTANCE-NAME>
INSTANCE_HOST: '127.0.0.1'
DB_PORT: 3306
DB_USER: <YOUR_DB_USER_NAME>
DB_PASS: <YOUR_DB_PASSWORD>
DB_NAME: <YOUR_DB_NAME>
```